generator/src/main/java/org/corfudb/generator/operations/NestedTxOperation.java

```diff
@@ -0,0 +1,64 @@
+package org.corfudb.generator.operations;
+
+import ch.qos.logback.classic.Logger;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.generator.State;
+import org.corfudb.runtime.exceptions.TransactionAbortedException;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+/**
+ * Created by rmichoud on 7/26/17.
+ */
+@Slf4j
+public class NestedTxOperation extends Operation {
+
+    Logger correctness = (Logger) LoggerFactory.getLogger("correctness");
+    private final static int maxNest = 20;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)